java/code/webapp/WEB-INF/pages/common/fragments/user/preferences.jspf

```diff
@@ -114,6 +114,30 @@
   </div>
 </div>
 
+<div class="panel panel-default">
```

Comment:
This is more of a question, and I understand this is probably historically very inconsistent, is aiming for semantically correct HTML a separate goal we try to address? For example, this would seem like a good candidate to be a `<section/>`.

Reply:
I would agree with a refactoring to improve the semantic, I believe we are at a very _low_ level at the moment, but probably it is better to set it as a target for another PR. I would be afraid in this case of CSS rules we have around those apply on single generic `<HTML>` tags. I give a :+1: to the aim, but let's do it separately.

Reply:
Sounds good, maybe we can create a card or a ticket to track this somewhere? This ties in tightly with accessibility as well.

Reply:
Agreed and done https://github.com/SUSE/spacewalk/issues/12770